src/api/alias/index.ts

```diff
@@ -10,9 +10,12 @@ import { DeleteApiRequestBuilder } from '../common/api-request-builder/delete-ap
 
 /**
  * Adds an alias to an existing note.
+ *
  * @param noteIdOrAlias The note id or an existing alias for a note.
  * @param newAlias The new alias.
  * @return Information about the newly created alias.
+ * @throws {Error} when the status code does not match the expected one or is defined as in the custom status code
+ *         error mapping.
  */
```

Comment:
This error description makes no sense here because none of the functions you changed in this file have an error mapping or an expected code. At least not from this call.

Reply:
Okay I see this a lot now. I'm not gonna mark this on every request. Please change that.